spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -48,15 +48,17 @@ <h1>Runtime Semantics: RegExpBuiltinExec ( _R_, _S_ )</h1>
         1. Perform ! CreateDataProperty(_A_, `"index"`, _lastIndex_).
         1. Perform ! CreateDataProperty(_A_, `"input"`, _S_).
         1. <del>Let _matchedSubstr_ be the matched substring (i.e. the portion of _S_ between offset _lastIndex_ inclusive and offset _e_ exclusive).</del>
-        1. <ins>Let _indices_ be the Record { [[Indices]]: &laquo; &raquo; }.</ins>
-        1. <ins>Let _match_ be the Match { [[StartIndex]]: _lastIndex_, [[EndIndex]]: _e_, [[GroupName]]: *undefined* }.</ins>
-        1. <ins>Add _match_ as the last element of _indices_.[[Indices]].</ins>
-        1. <ins>Let _matchedValue_ be ! GetMatchString(_S_, _match_)</ins>
+        1. <ins>Let _indices_ be &laquo; &raquo;.</ins>
+        1. <ins>Let _match_ be the Match { [[StartIndex]]: _lastIndex_, [[EndIndex]]: _e_ }.</ins>
+        1. <ins>Add _match_ as the last element of _indices_.</ins>
+        1. <ins>Let _matchedValue_ be ! GetMatchString(_S_, _match_).</ins>
         1. Perform ! CreateDataProperty(_A_, `"0"`, <del>_matchedSubstr_</del><ins>_matchedValue_</ins>).
         1. If _R_ contains any |GroupName|, then
           1. Let _groups_ be ObjectCreate(*null*).
+          1. <ins>Let _groupNames_ be &laquo; &raquo;.</ins>
```

Comment:
```suggestion
          1. <ins>Let _groupNames_ be a new empty List.</ins>
```

Reply:
Why? https://tc39.es/ecma262/#sec-list-and-record-specification-type specifies:

> A new empty List can be expressed as &laquo; &raquo;.

It's used this way in 3 other places in the spec so far: 
- [Runtime Semantics: Labeled Evaluation (Step 10)](https://tc39.es/ecma262/#sec-for-statement-runtime-semantics-labelledevaluation)
- [Runtime Semantics: Case Block Evaluation (Step 3.a)](https://tc39.es/ecma262/#sec-runtime-semantics-caseblockevaluation)
- [Runtime Semantics: Case Block Evaluation (Step 8.a)](https://tc39.es/ecma262/#sec-runtime-semantics-caseblockevaluation)

Reply:
More for consistency; the phrase "new empty list" appears 169 times in the spec. We should instead probably clean up those 3 outliers rather than having two ways to express it.